op-chain-ops/cmd/registry-data/main.go

```diff
@@ -209,18 +211,20 @@ func entrypoint(ctx *cli.Context) error {
 
 	// convert into allocation data
 	out := Genesis{
-		Nonce:      genesis.Nonce,
-		Timestamp:  genesis.Timestamp,
-		ExtraData:  genesis.ExtraData,
-		GasLimit:   genesis.GasLimit,
-		Difficulty: (*hexutil.Big)(genesis.Difficulty),
-		Mixhash:    genesis.Mixhash,
-		Coinbase:   genesis.Coinbase,
-		Number:     genesis.Number,
-		GasUsed:    genesis.GasUsed,
-		ParentHash: genesis.ParentHash,
-		BaseFee:    (*hexutil.Big)(genesis.BaseFee),
-		Alloc:      make(jsonutil.LazySortedJsonMap[common.Address, GenesisAccount]),
+		Nonce:         genesis.Nonce,
+		Timestamp:     genesis.Timestamp,
+		ExtraData:     genesis.ExtraData,
+		GasLimit:      genesis.GasLimit,
+		Difficulty:    (*hexutil.Big)(genesis.Difficulty),
+		Mixhash:       genesis.Mixhash,
+		Coinbase:      genesis.Coinbase,
+		Number:        genesis.Number,
+		GasUsed:       genesis.GasUsed,
+		ParentHash:    genesis.ParentHash,
+		BaseFee:       (*hexutil.Big)(genesis.BaseFee),
+		ExcessBlobGas: genesis.ExcessBlobGas, // EIP-4844
+		BlobGasUsed:   genesis.BlobGasUsed,   // EIP-4844
```

Comment:
Because these fields were missing, the data being stored in `superchain-registry` has been incomplete.